docs/docs/user/core-concepts/config.md

```diff
@@ -0,0 +1,22 @@
+# Configuration
+
+KPOps reads its global configuration that is unrelated to a pipeline's
+[components](./components/overview.md) from [`config.yaml`](.#__codelineno-0-1).
```

Comment:
This seems to fix the warning for me. Does it work for you, too?

```suggestion
[components](./components/overview.md) from [`config.yaml`](#__codelineno-0-1).
```

Reply:
Yes this removed the warning!